tests/unit/drm_plugin_test.sh

```diff
@@ -332,6 +363,39 @@ function test_get_supported_mode_per_connector()
   export SYSFS_CLASS_DRM="$FAKE_DRM_SYSFS"
   output=$(get_supported_mode_per_connector 2)
   compare_command_sequence '' "$LINENO" 'expected_output' "$output"
+
+  declare -a expected_output=(
```

Comment:
Replace all double quotes with single quotes where we don't have a bash variable.